### PR TITLE
EPMRPP-38865 || Pattern analysis activities api

### PR DESCRIPTION
**Pull request by Yumfriez**

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -72,7 +72,7 @@ rp:
   db:
     url: jdbc:postgresql://${rp.db.host}:${rp.db.port}/${rp.db.name}
     name: reportportal
-    host: postgres
+    host: localhost
```
Comment (pbortnik) [inline src/main/resources/application.yaml]: change back

File: src/main/resources/datastore.properties [hunk 2]
```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-datastore.default.path=${rp.binarystore.path:/data/storage}
+datastore.default.path=${rp.binarystore.path:/Users/yumfriez/data/storage}
```
Comment (pbortnik) [inline src/main/resources/datastore.properties]: change back

Comment (Yumfriez) [reply src/main/resources/datastore.properties]: fixed

File: src/main/resources/application.yaml (hunk 1 above)
Comment (Yumfriez) [reply src/main/resources/application.yaml]: fixed

**Pull request merged**